packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -209,9 +209,23 @@ export default class Editor {
      * @returns true if node is replaced. Otherwise false
      */
     public replaceNode(existingNode: Node, toNode: Node): boolean {
+        // Transform the new node to replace to
+        const darkModeOptions = this.getDarkModeOptions();
+        const darkModeTransform = this.isDarkMode()

```

Comment:
This changes the existing behavior. Some customer may rely on the existing behavior to not convert replaced node to dark mode. 
Consider add a parameter to control this behavior

Reply:
Got it, a param has been added.